### ISSUE TITLE
DeCLI - Move schema-to-cli functionality to a new project

### ISSUE THREAD

**Issue by mickmcgrath13** [bitovi/bitops#230]
**Describe the solution you'd like**
DeCLI would be a tool that takes a yaml schema and a config file and outputs a cli command.

Steps:
- extract the schema logic from bitops (post V2 release)
- move code to its own repo/project - ideally with a CI process that outputs a binary
- integrate the DeCLI binary into bitops

**Additional context**
More details here: https://github.com/bitovi/bitops/issues/59

**Comment by armab** [comment]
Is the current code available as bash here https://github.com/bitovi/bitops/tree/master/scripts/bitops-config and the idea to convert it to some standalone CLI tool?

**Comment by mickmcgrath13** [comment]
@armab yes.  V2 has its an updated implementation in [plugins/utilities.py](https://github.com/bitovi/bitops/blob/plugins/scripts/plugins/utilities.py) that we'd probably want to work off of because it's python instead of bash.
